extensions/cms/src/test/cmsResource/tree/treeProvider.test.ts

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as vscode from 'vscode';
+import * as should from 'should';
+import * as TypeMoq from 'typemoq';
+import 'mocha';
+import { AppContext } from '../../../appContext';
+import { ApiWrapper } from '../../../apiWrapper';
+
+import { CmsResourceTreeProvider } from '../../../cmsResource/tree/treeProvider';
+import { CmsResourceMessageTreeNode } from '../../../cmsResource/messageTreeNode';
+import { CmsResourceEmptyTreeNode } from '../../../cmsResource/tree/cmsResourceEmptyTreeNode';
+import { CmsResourceTreeNode } from '../../../cmsResource/tree/cmsResourceTreeNode';
+
+// Mock services
+let mockAppContext: AppContext;
+let mockExtensionContext: TypeMoq.IMock<vscode.ExtensionContext>;
+let mockApiWrapper: TypeMoq.IMock<ApiWrapper>;
+
+
+describe('CmsResourceTreeProvider.getChildren', function (): void {
+	beforeEach(() => {
+		mockExtensionContext = TypeMoq.Mock.ofType<vscode.ExtensionContext>();
+		mockApiWrapper = TypeMoq.Mock.ofType<ApiWrapper>();
+		mockAppContext = new AppContext(mockExtensionContext.object, mockApiWrapper.object);
+	});
+
+	it('Should not be initialized.', async function (): Promise<void> {
+		const treeProvider = new CmsResourceTreeProvider(mockAppContext);
+		should.notEqual(treeProvider.isSystemInitialized, true);
```

Comment:
Should also test this after the call to getChildren to ensure it's still the expected value afterwards